quipucords/client/index.html

```diff
@@ -1,75 +1,84 @@
 <!DOCTYPE html>
-<!--[if IE 9]><html lang="en-us" class="ie9 login-pf"><![endif]-->
-<!--[if gt IE 9]><!--><html lang="en-us" class="login-pf"><!--<![endif]-->
+<!--[if IE 9]><html lang="en-us" class="ie9 layout-pf layout-pf-fixed"><![endif]-->
+<!--[if gt IE 9]><!--><html lang="en-us" class="layout-pf layout-pf-fixed"><!--<![endif]-->
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title>Patternfly Enterprise Application</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Red Hat Entitlements Reporting</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/css/patternfly.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/css/patternfly-additions.min.css">
+    <link rel="shortcut icon" href="favicon.ico">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.37.9/css/patternfly.min.css">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.37.9/css/patternfly-additions.min.css">
 
     <script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
-    <script src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/js/patternfly.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.37.9/js/patternfly.min.js"></script>
   </head>
   <body>
-    <span id="badge">
-      <img src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/img/logo.svg" alt="PatternFly logo" />
-    </span>
+    <nav class="navbar navbar-pf-vertical">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a href="/" class="navbar-brand">
+          <img class="navbar-brand-icon" src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/img/logo-alt.svg" alt=""/><img class="navbar-brand-name" src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/img/brand-alt.svg" alt="PatternFly Enterprise Application" />
+        </a>
+      </div>
+      <nav class="collapse navbar-collapse">
+        <ul class="nav navbar-nav navbar-right navbar-iconic navbar-utility">
 
```

Comment:
What is all the nav stuff here?

Reply:
Most of this stuff is just placeholder. Copied it from  Patternfly site.